CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
```

Comment:
Yes, updating the changelog sounds good. I'd even suggest that every change
in master should be added to the changelog immediately under a "unreleased"
or similar heading. Ensures that changes get documented and people can see
what the difference of master to the last released version is. Will post
examples from other projects later.
On May 21, 2015 17:55, "Julius Volz" <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/prometheus/prometheus/pull/715#discussion_r30852066>:
>
> > @@ -1,3 +1,20 @@
> > +## 0.14.0-rc1 / 2015-05-21
>
> @grobie <https://github.com/grobie> So far, RC releases have been kind of
> under the radar, for internal early canarying. This time we actually posted
> an RC to the mailing list for the first time, so it might actually make
> sense to update the changelog earlier, yeah. I'd be fine with adding it to
> the Changelog, but then updating the existing entry upon the full release
> instead of adding yet another one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/715/files#r30852066>.
>